docs/source/examples/notebooks/parameterization/nmc_pouch_cell_BPX.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
should this be moved to [pybamm-data](https://github.com/pybamm-team/pybamm-data)? how do you actually add things there?

Reply:
This is a pretty small file, so there is not a major issue with putting it in the repo. Having the data on pybamm-data will probably make it more compatible with the google-colab functionality in the documentation

Reply:
> should this be moved to [pybamm-data](https://github.com/pybamm-team/pybamm-data)?

> This is a pretty small file, so there is not a major issue with putting it in the repo. Having the data on pybamm-data will probably make it more compatible with the google-colab functionality in the documentation

Yes, I think we should centralise everything under [`pybamm-data`](https://github.com/pybamm-team/pybamm-data/releases/tag/v1.0.0), so it would be nice to add this file there regardless of its small size (we do have a user-facing `DataLoader` class, after all, so it makes sense to showcase it more through more example notebooks).

> how do you actually add things there?

The procedure for adding things there is to first create a new v1.0.1 release and add all of the files from the previous release, add this file, and then bump the version in `pybamm_data.py`. We do this for reproducibility, such that the next PyBaMM release will retrieve the files from a different release URL (and this gives us the added benefit of better-segregated download analytics/usage stats from the GitHub API). If the changes to this file are final, I can do this later today if you'd like, @rtimms, or step back in case you may wish to do this yourself; please let me know!

Reply:
done, thanks!

Reply:
@agriyakhetarpal Can you add documentation to the pybamm-data repo for how that is done?